### PR TITLE
[13.0][MIG]product exception: Migration to 13.0

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
@ForgeFlow

### CODE REVIEW

File: product_exception/README.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+Configuration
+=============
+
+This module don't provide any menu to edit the rules.
+See base_exception for more information on rules.
+
+Usage
+=====
+
+A button "Check Exception" is present on the header of the product form.
```
Comment (mariadforgeflow) [inline product_exception/README.rst:49]: Just a minor comment, the button now is called "test button"

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

File: product_exception/README.rst (hunk 1 above)
Comment (GuillemCForgeFlow) [reply product_exception/README.rst:49]: Thanks! :smile:

File: product_exception/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Quality",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["hparfr"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["stock", "base_exception"],
```
Comment (MiquelRForgeFlow) [inline product_exception/__manifest__.py:13]:
```suggestion
    "depends": ["product", "base_exception"],
```
?, not sure, but I don't see in the code why it should depend from stock

Comment (GuillemCForgeFlow) [reply product_exception/__manifest__.py:13]: You're right, it must depend on the product as it's inheriting the views from the module. There's no relation with the stock module.

File: product_exception/views/product.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            name="context"
+        >{'active_test': False, 'default_model' : 'product.template'}</field>
+    </record>
+
+    <menuitem
+        id="menu_product_test_parent"
+        name="Product Exceptions Rules"
+        action="action_product_test_tree"
+        sequence="90"
+        parent="stock.menu_product_in_config_stock"
```
Comment (MiquelRForgeFlow) [inline product_exception/views/product.xml:22]: yes, stock is needed :S

sorry for the noise

Comment (GuillemCForgeFlow) [reply product_exception/views/product.xml:22]: It's okay :) I didn't see it at first

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: product_exception/__manifest__.py (hunk 2 above)
Comment (etobella) [inline product_exception/__manifest__.py:13]: Can you change it to "product", please?

**Review by etobella — CHANGES_REQUESTED** [review]

File: product_exception/__manifest__.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply product_exception/__manifest__.py:13]: Selection_689

Comment (etobella) [reply product_exception/__manifest__.py:13]: Then, it is incongruent with the README :laughing: 
https://github.com/OCA/product-attribute/pull/1038/files#diff-6fd6b0c73055eccb34ab510b26f4e4ca5d07e189b8f778236a9970743e63ec90R40-R44

**Comment by rousseldenis** [comment]
@GuillemCForgeFlow @LoisRForgeFlow @MiquelRForgeFlow

File: product_exception/__manifest__.py (hunk 2 above)
Comment (GuillemCForgeFlow) [reply product_exception/__manifest__.py:13]: I've removed the sentence from the README file, the module needs to add the menu item to manage exception rules on products.

**Comment by GuillemCForgeFlow** [comment]
@rousseldenis Should be ready now

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis ready to retry

**Pull request merged**